sonic_sfp/sfputilbase.py

```diff
@@ -198,7 +198,7 @@ def _sfp_eeprom_present(self, sysfs_sfp_i2c_client_eeprompath, offset):
             return False
         else:
             try:
-                sysfsfile = open(sysfs_sfp_i2c_client_eeprompath, "rb")
```

Comment:
Suggest open with 'with'.
[https://www.pythonforbeginners.com/files/with-statement-in-python](https://www.pythonforbeginners.com/files/with-statement-in-python)

Reply:
Done changing this place. thanks!